packages/toaster/src/toast.tsx

```diff
@@ -149,26 +85,30 @@ export const Toast: React.FC<Props> = ({children, ...props}) => {
       onExited={defaults.onRemove}
     >
       {state => (
-        <WrapperAnimated
+        <ToastWrapperAnimated
           state={state}
           onMouseEnter={handleMouseEnter}
           onMouseLeave={handleMouseLeave}
           style={{
             height,
-            zIndex: defaults.zIndex,
-            marginBottom: isShown ? 0 : -height
+            zIndex: defaults.zIndex
           }}
         >
-          <div ref={onRef} style={{padding: 8}}>
-            <Alert
-              appearance="card"
+          <ToastWrapper ref={onRef} hasCloseButton={defaults.hasCloseButton}>
+            {defaults.hasCloseButton && (
```

Comment:
Move this close icon directly into `@smashing/alert`. Make it hidden by default.